src/core/agent/services/credentialService.ts

```diff
@@ -69,14 +69,19 @@ class CredentialService extends AgentService {
   }
 
   @OnlineOnly
-  async getCredentialDetailsById(id: string): Promise<ACDCDetails> {
+  async getCredentialDetailsById(
+    id: string
+  ): Promise<Omit<ACDCDetails, "credentialType" | "issuanceDate">> {
     const metadata = await this.getMetadataById(id);
     const acdc = await this.props.signifyClient.credentials().get(metadata.id);
     if (!acdc) {
       throw new Error(CredentialService.CREDENTIAL_NOT_FOUND);
     }
+    const credentialShortDetails = this.getCredentialShortDetails(metadata);
     return {
-      ...this.getCredentialShortDetails(metadata),
+      id: credentialShortDetails.id,
+      schema: credentialShortDetails.schema,
```

Comment:
Is this actually used by the UI? We have the `s` field

Reply:
I see we are using this field to check that the credential is RareEvo, Can we use s.title?

Reply:
Ah, no, then we keep it. I forgot it was used for that. The SAID is the only unique element of a schema so needs to be this. Nevermind